main/webapp/modules/core/scripts/dialogs/expression-preview-dialog.js

```diff
@@ -195,6 +234,14 @@ ExpressionPreviewDialog.Widget.prototype._getLanguage = function() {
     return this._elmts.expressionPreviewLanguageSelect[0].value;
 };
 
+ExpressionPreviewDialog.Widget.prototype._getCode = function() {
+    return jQueryTrim(this._elmts.expressionPreviewTextarea[0].value || "");
+};
+
+ExpressionPreviewDialog.Widget.prototype._getExpression = function() {
```

Comment:
Since there's already a method called `getExpression` with a different purpose from this one, I think it would be helpful to rename this to avoid any confusion.